src/ServicePulse.Host/ServicePulse.Host.csproj

```diff
@@ -36,4 +36,10 @@
     <EmbeddedResource Include="Hosting\Help.txt" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Does this need to be explicitly added? I think it should automatically get embedded based on line 31.

Reply:
@WilliamBZA strange.. that must be removed....Pretty sure I did that